src/main/java/com/db/crud/voting/service/impl/UserServiceImpl.java

```diff
@@ -55,11 +57,19 @@ private void verifyUserPresent(Optional<User> user) {
     }
 
     @Override
-    public UserResponse getUser(String cpf) {
+    public UserResponse getUserResponse(String cpf) {
+        log.info("Requested Specific User!");
         User user = userRepository.findByCpf(cpf).orElseThrow(() -> new CannotFindEntityException("Cannot find this User"));
         return userMapper.userToDto(user);
     }
 
+    @Override
+    public User getUser(String cpf) {
+        return userRepository.findByCpf(cpf).orElseThrow(
+            () -> new CannotFindEntityException("The user with cpf: "+cpf+" isn't registered!")
+        );
+    }
+
     private LogObj buildObj(String type, Long id, String name, Operation operation, LocalDateTime realizedOn) {
         return logMapper.logObj(type, id, name, operation, realizedOn);
```

Comment:
```suggestion
    private LogObj buildObj(String type, Long id, String name, Operation operation, LocalDateTime realizedOn) {
        return logMapper.logObj(type, id, name, operation, realizedOn);
```

Notei que você repete esse método no service de Agenda. Talvez fosse mais interessante tornar o método público e mover ele para o LogService. Dessa forma, você poderia centralizar a lógica e também mover a dependência LogMapper, que também está sendo repetida no service de Agenda e Usuário, assim vc teria um código reutilizável e mais fácil de dar manutenção no futuro.

Reply:
De fato, esse método se repete desnecessariamente nessas Services. Implementei ele na logService e chamei por lá.

Commit: [Move LogObj to logService and authorizeAdmin to userService](https://github.com/AurorinhaBoreal/VotacaoCRUD-BackEnd/pull/37/commits/e548d5696fe68876cebb85321e63d89fb196b58b)